app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def select_type(skill_type)
+    @character.skills.order(:name).select{ |type_skill| type_skill.type_of_skill == skill_type }
```

Comment:
@kurenn @Anapaotrev I agree with adding a scope `by_name` onto `Character` model, but I guess we can get ride of the `select` also with another scope:


`by_skill` scope on `Character` model will be pretty much the same

```ruby
scope :by_skill, (skill_name) -> { skills.where(name: skill_name) }
```
And your helper can be reduced to:

```ruby
def select_type character, skill_type
  character.by_skill(skill_type).by_name
end
```

What do you think? 🤔